### ISSUE TITLE
error : thread 'main' panicked at 'attempt to calculate the remainder with a divisor of zero'

### ISSUE THREAD

**Issue by SansGuidon** [njaremko/podcast#11]
After building podcast locally on MacOS, I've attempted to use it on a couple of podcast feeds I'm subscribed to, and I got the same issue every time 

```
❯ podcast subscribe https://legacycoderocks.libsyn.com/rss
Using PODCAST dir: "/Users/morgangeek/Podcasts"
Creating new file: "/Users/morgangeek/Podcasts/.subscriptions.json"
thread 'main' panicked at 'attempt to calculate the remainder with a divisor of zero', /Users/morgangeek/.cargo/registry/src/github.com-1ecc6299db9ec823/bloom-0.3.2/src/bloom.rs:165:23
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

❯ podcast subscribe https://feeds.simplecast.com/7y1CbAbN
Using PODCAST dir: "/Users/morgangeek/Podcasts"
Creating new file: "/Users/morgangeek/Podcasts/.subscriptions.json"
thread 'main' panicked at 'attempt to calculate the remainder with a divisor of zero', /Users/morgangeek/.cargo/registry/src/github.com-1ecc6299db9ec823/bloom-0.3.2/src/bloom.rs:165:23
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

❯ podcast subscribe https://therabbithole.libsyn.com/rss
Using PODCAST dir: "/Users/morgangeek/Podcasts"
Creating new file: "/Users/morgangeek/Podcasts/.subscriptions.json"
thread 'main' panicked at 'attempt to calculate the remainder with a divisor of zero', /Users/morgangeek/.cargo/registry/src/github.com-1ecc6299db9ec823/bloom-0.3.2/src/bloom.rs:165:23
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

When adding the `RUST_BACKTRACE=full` : 

```
❯ RUST_BACKTRACE=full podcast subscribe https://therabbithole.libsyn.com/rss
Using PODCAST dir: "/Users/morgangeek/Podcasts"
Creating new file: "/Users/morgangeek/Podcasts/.subscriptions.json"
thread 'main' panicked at 'attempt to calculate the remainder with a divisor of zero', /Users/morgangeek/.cargo/registry/src/github.com-1ecc6299db9ec823/bloom-0.3.2/src/bloom.rs:165:23
stack backtrace:
   0:        0x108ad9d2e - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::he3a6eecb2201efb0
   1:        0x108b0399c - core::fmt::write::h8ef98027ac1df1be
   2:        0x108ad36f7 - std::io::Write::write_fmt::hed52d76b85d70756
   3:        0x108adc1d5 - std::panicking::default_hook::{{closure}}::h013242fa408d10b7
   4:        0x108adbf12 - std::panicking::default_hook::h9777ceea989331ac
   5:        0x108adc86d - std::panicking::rust_panic_with_hook::h845f7f3fcb3a3323
   6:        0x108adc412 - rust_begin_unwind
   7:        0x108b015ef - core::panicking::panic_fmt::h60d70f2d73364aac
   8:        0x108b01547 - core::panicking::panic::h3582cc3308948195
   9:        0x10874e3c4 - <core::future::from_generator::GenFuture<T> as core::future::future::Future>::poll::hcc62750940782703
  10:        0x108757a6a - podcast::main::{{closure}}::hf4b38dd69138f904
  11:        0x1086fd5ab - blocking::block_on::hf5a086a9e76ecbef
  12:        0x108727355 - tokio::runtime::context::enter::h0da0e469e374495a
  13:        0x1086ee9e4 - std::thread::local::LocalKey<T>::with::hbe0a4c293eae0fcc
  14:        0x1087164f5 - podcast::main::h12466b92bb2f9233
  15:        0x10872dc8b - std::rt::lang_start::{{closure}}::ha47d1e2bc3c1e5bf
  16:        0x108adcbf9 - std::rt::lang_start_internal::hf7eb5bf96c0eb063
  17:        0x108716559 - main
```

Current version of podcast and rust tools : 
```
❯ podcast --version
podcast 0.17.4
❯ rustup --version
rustup 1.20.2 (2019-11-03)
❯ rustc --version
rustc 1.45.0 (5c1f21c3b 2020-07-13)
```

thanks in advance for your help ! I'm really looking forward to try this tool for my needs :-)

**Comment by njaremko** [comment]
Thanks for letting me know, would have taken a while to notice this since it requires no existing subscriptions.

Fixed in `0.17.5`

**Comment by SansGuidon** [comment]
Thank you for the fast resolution @njaremko :)
